react/fields/MonthPicker/CustomMonthPicker/CustomMonthPicker.js

```diff
@@ -0,0 +1,168 @@
+import styles from './CustomMonthPicker.less';
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import range from 'lodash.range';
+
+import Dropdown from '../../Dropdown/Dropdown';
+
+const getYearOptions = () => {
+  const maxYear = new Date().getFullYear();
+  const minYear = maxYear - 100;
+
+  return range(maxYear, minYear - 1).map(value => {
+    const stringValue = `${value}`;
```

Comment:
Yeah that's probably better as it's more declarative what's it's intending to do